### PR TITLE
Added SPT validation with DL server

### PR DESCRIPTION
**Pull request by chebroluharika**
### Description
Added SPT validation with DL server

### Issues Resolved
NA

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by nabhajit-ray — COMMENTED** [review]
We do not need check for enclosure group for DL while updating? Line number 2491?

File: oneview/resource_server_profile_template.go [hunk 1]
```diff
@@ -2482,13 +2487,14 @@ func resourceServerProfileTemplateUpdate(d *schema.ResourceData, meta interface{
...
-		return err
+	if d.Get("enclosure_group") != "" {
+		val := d.Get("enclosure_group")
+		enclosureGroup, err := config.ovClient.GetEnclosureGroupByName(val.(string))
+		if err != nil {
+			return err
+		}
+		serverProfileTemplate.EnclosureGroupURI = enclosureGroup.URI
 	}
-	serverProfileTemplate.EnclosureGroupURI = enclosureGroup.URI
```
Comment (AvijitOS) [inline oneview/resource_server_profile_template.go:2491]: i think this line should be inside if condition as done above

**Review by AvijitOS — COMMENTED** [review]
changes look ok to me

**Review by nabhajit-ray — APPROVED** [review]
LGTM

**Review by AvijitOS — APPROVED** [review]

File: oneview/resource_server_profile_template.go (hunk 1 above)
Comment (chebroluharika) [reply oneview/resource_server_profile_template.go:2491]: hey, its inside if block only

**Pull request merged**